spec/extra/format/date-time.json

```diff
@@ -0,0 +1,53 @@
+[
+    {
```

Comment:
since we're adding support for `format` there should also be a test-case for an "unknown" format (which means it must always pass validation)